.github/workflows/test.yaml

```diff
@@ -179,14 +171,14 @@ jobs:
 
       # Installing fixed version of Chrome since latest version does not work (128 didn't work)
       # Leaving this here again in case we need to pin to a specific Chrome version in the future
-      - name: Install Chrome 127
-        run: |
-          sudo apt-get install -y wget libu2f-udev
-          cd /tmp
-          wget -q http://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_127.0.6533.72-1_amd64.deb
-          sudo dpkg -i google-chrome-stable_127.0.6533.72-1_amd64.deb
-          sudo apt-get install -y --allow-downgrades ./google-chrome-stable_127.0.6533.72-1_amd64.deb
-          google-chrome --version
+      # - name: Install Chrome 127
```

Comment:
remove un-needed chrome pin
- this commented out section was a c+p from the main e2e section
- the main e2e section comments this out
- both should use a single repo gh action, won't do that now though

Reply:
this has now been done upstream